### ISSUE TITLE
The subscriber TFSAggregator raised an exception while being notified of event

### ISSUE THREAD

**Issue by alexander-lysenko-88** [Vaccano/TFS-Aggregator#15]
Hi, 
I'm trying to use you Plugin and faced with a problem. After installation it on TFS Server "Basic test" doesn't work and I can see such error in Windows Event Log.

...
Process Details:
  Process Name: w3wp
  Process Id: 6316
  Thread Id: 11896
  Account name: NT AUTHORITY\NETWORK SERVICE

Detailed Message: The subscriber TFSAggregator raised an exception while being notified of event Microsoft.TeamFoundation.WorkItemTracking.Server.WorkItemChangedEvent.
Exception Message: The source was not found, but some or all event logs could not be searched.  To create the source, you need permission to read all event logs to make sure that the new source name is unique.  Inaccessible logs: Security. (type SecurityException)
Exception Stack Trace:    at System.Diagnostics.EventLog.FindSourceRegistration(String source, String machineName, Boolean readOnly, Boolean wantToCreate)
   at System.Diagnostics.EventLog.SourceExists(String source, String machineName, Boolean wantToCreate)
   at System.Diagnostics.EventLogInternal.VerifyAndCreateSource(String sourceName, String currentMachineName)
   at System.Diagnostics.EventLogInternal.WriteEntry(String message, EventLogEntryType type, Int32 eventID, Int16 category, Byte[] rawData)
   at System.Diagnostics.EventLog.WriteEntry(String source, String message, EventLogEntryType type, Int32 eventID, Int16 category, Byte[] rawData)
   at System.Diagnostics.EventLog.WriteEntry(String source, String message, EventLogEntryType type)
   at TFSAggregator.MiscHelpers.LogMessage(String message, Boolean isError)
   at TFSAggregator.WorkItemChangedEventHandler.ProcessEvent(TeamFoundationRequestContext requestContext, NotificationType notificationType, Object notificationEventArgs, Int32& statusCode, String& statusMessage, ExceptionPropertyCollection& properties)
   at Microsoft.TeamFoundation.Framework.Server.TeamFoundationEventService.SubscriptionList.Notify(TeamFoundationRequestContext requestContext, NotificationType notificationType, Object notificationEventArgs, String& statusMessage, ExceptionPropertyCollection& properties)
...

Could you please help me with this issue?

Best regards,
Alexander

**Comment by giuliov** [comment]
There is a bug in the install instructions. You must create an Source named `TFSAggregator` in the Application Event Log.
It is one Powershell line to run where the Aggregator is installed.

```
New-EventLog -LogName "Application" -Source "TFSAggregator"
```

Note that only errors are written in Windows Event Log, so this is a symptom of something else.

**Comment by giuliov** [comment]
Corrected with PR #17

**Comment by alexander-lysenko-88** [comment]
Thank you!
